src/components/EditProfile.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class EditProfile extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      email: '',
+      bio: '',
+      firstName: '',
+      lastName: ''
+    }
+  }
+
+  submitForm() {
+    // go through items in the state and only send the ones that have been updated
+    // make sure that the user can only submit an edit request for their OWN user account
+  }
+
+// Kash was here
+
+  render() {
+    return(
+      <div className="row">
+        <div className="col-md-2"></div>
+        <div className="col-md-8">
+          <div className="card">
+            <div className="card-header card-header-icon" data-background-color="rose">
+              <i className="material-icons">perm_identity</i>
+            </div>
+            <div className="card-content">
+              <h4 className="card-title">Edit Profile  -
+                <small className="category">  Complete your profile</small>
+              </h4>
+              <form>
+                <div className="row">
+                  <div className="col-md-3">
+                    <div className="form-group label-floating is-empty">
+                      <label className="control-label">Username</label>
+                      <input type="text" className="form-control" value={this.state.username} onChange={(e) => this.setState({username: e.target.value})}/>
```

Comment:
Why not deconstruct all the state variables above the return?...
`const { username, email, firstName, lastName, bio, userId } = this.state`